osgar/drivers/kloubak.py

```diff
@@ -80,8 +78,8 @@ def draw_enc_stat(arr):
 
 def compute_desired_erpm(desired_speed, desired_angular_speed):
     scale = 1 / (VESC_REPORT_FREQ * SPEED_ENC_SCALE)
-    left = scale * (desired_speed - desired_angular_speed * WHEEL_DISTANCE / 2)
-    right = scale * (desired_speed + desired_angular_speed * WHEEL_DISTANCE / 2)
+    left = scale * (desired_speed - desired_angular_speed * wheel_distance / 2)
```

Comment:
here I was wondering where it is coming from? I would use g_wheel_distance for "Global" or the original "constants" with note, that they can be modified by config??